extensions/src/Smithy.Identity.Abstractions/Smithy.Identity.Abstractions.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Add the .NET Framework targets as I mentioned above so we can do conditional compilations for .NET Framework.

Reply:
Changed the target frameworks in https://github.com/aws/aws-sdk-net/pull/3330/commits/8d996a29a697a419323c524209c5af250b4d5280 (using `net462` as this is the current value in the `v4` branch, will update it as soon we make the switch to `net472`).